api/turing/cluster/servicebuilder/service_builder.go

```diff
@@ -85,10 +85,6 @@ type ClusterServiceBuilder interface {
 		secretName string,
 		fluentdConfig *config.FluentdConfig,
 	) *cluster.KubernetesService
-	NewPluginsServerService(
```

Comment:
I think this method still features in `api/turing/service/router_deployment_service_test.go` and possibly elsewhere - we can clean it up.

Reply:
Whoops, okay let me go hunt them down 😶